src/extensions/extension-manager.ts

```diff
@@ -74,31 +79,36 @@ export class ExtensionManager {
   }
 
   async loadExtensions() {
-    const extensions = await this.loadFromFolder(this.folderPath);
+    const extensions = await this.loadFromFolder([this.inTreeFolderPath, this.localFolderPath])
```

Comment:
We need to separate in-tree & local-folder loading because they behave a bit differently (local-folder extensions are enabled by default and they can be enabled/disabled when we have UI available).

Reply:
Changed. PTAL.